README.md

```diff
@@ -108,6 +107,28 @@ ansible-playbook -i ./BraveButtonsConfig/ansible/<inventory file name> \
                  --ask-vault-pass
 ```
 
+# How to release a new version & update the entire Button system (server and RPi fleet)
+
+1. decide on an appropriate version number for the new version
+
+1. update CHANGELOG.md by moving everything in `Unreleased` to a section for the new version
+
+1. make a new git tag in both the `BraveButtons` and `BraveButtonsConfig` repositories with the version number
```

Comment:
Could you please include an example git command for this just to make sure that we're always tagging in the same way?